app/templates/main.php

```diff
@@ -0,0 +1,65 @@
+<section class="content__side">
+    <h2 class="content__side-heading">Проекты</h2>
+
+    <nav class="main-navigation">
+        <ul class="main-navigation__list">
+            <?php foreach ($projects as $project): ?>
+                <li class="main-navigation__list-item">
+                    <a class="main-navigation__list-item-link" href="#"><?= htmlspecialchars($project); ?></a>
```

Comment:
; в конце необязательны в данном случае, лучше убрать для красоты)
```suggestion
                    <a class="main-navigation__list-item-link" href="#"><?= htmlspecialchars($project) ?></a>
```

Reply:
Для красоты расставлял :) Ок, уберу.